app/types/transport/requests/status/create-status.ts

```diff
@@ -0,0 +1,21 @@
+
+import { NewMessage } from 'ctproto';
+
+export interface CreateStatusPayload {
```

Comment:
We don't have the client side for status creation. Why do we have this interface?

Reply:
had to add method to create predefined statuses when new project is created
https://github.com/codex-team/task-manager/pull/104/files#diff-625fbfe36c02ae640152af5f420ec1e9985286c791177a7cf1b65418a87ec674R17